cmd/kops/rollingupdatecluster.go

```diff
@@ -41,49 +43,75 @@ type RollingUpdateClusterCmd struct {
 	NodeInterval    time.Duration
 	BastionInterval time.Duration
 
-	cobraCommand *cobra.Command
+	ClusterName string
+
+	// InstanceGroups is the list of instance groups to rolling-update;
+	// if not specified all instance groups will be updated
+	InstanceGroups []string
+}
+
+func (o *RollingUpdateOptions) InitDefaults() {
+	o.Yes = false
+	o.Force = false
+	o.CloudOnly = false
+
+	o.MasterInterval = 5 * time.Minute
+	o.NodeInterval = 2 * time.Minute
+	o.BastionInterval = 5 * time.Minute
```

Comment:
We can probably turn this down - bastions literally do nothing but run an SSH daemon